### PR TITLE
Chips

### PR DESCRIPTION
**Pull request by chowington**

### CODE REVIEW

File: src/lib/core/components/FilterChipList.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    variable.id === props.selectedVariableId
+                  }
+                  // Remove this filter on click of X button
+                  onDelete={() =>
+                    props.sessionState.setFilters(
+                      session.filters.filter((f) => f !== filter)
+                    )
+                  }
+                />
+              </VariableLink>
```
Comment (dmfalke) [inline src/lib/core/components/FilterChipList.tsx:88]: Try passing the `VariableLink` element to the `text` prop. This should address the issue of the page reloading when clicking the delete button. You will probably need to modify the link text color.
```suggestion
                <FilterChip
                  text={(
                    <VariableLink
                      entityId={entity.id}
                      variableId={variable.id}
                      replace={true}
                      key={`filter-chip-${entity.id}-${variable.id}`}
                  >
                    {variable.displayName}
                   </VariableLink>
                  )}
                  tooltipText={filterValueDisplay}
                  isActive={
                    entity.id === props.selectedEntityId &&
                    variable.id === props.selectedVariableId
                  }
                  // Remove this filter on click of X button
                  onDelete={() =>
                    props.sessionState.setFilters(
                      session.filters.filter((f) => f !== filter)
                    )
                  }
                />
```

**Review by dmfalke — CHANGES_REQUESTED** [review]
This is looking really good. See my suggestions inline.

File: src/lib/core/components/FilterChip.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Chip } from '@material-ui/core';
+
+interface Props {
+  text: string;
+  tooltipText: string;
+  isActive: boolean;
+  onClick?: () => void;
```
[the remark this answers is not in the document]
Comment (chowington) [reply src/lib/core/components/FilterChip.tsx]: Nope!

File: src/lib/core/components/FilterChip.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { Chip } from '@material-ui/core';
+
+interface Props {
+  text: string;
```
[the remark this answers is not in the document]
Comment (chowington) [reply src/lib/core/components/FilterChip.tsx]: Inner content is passed as a React node now 👍

**Review by dmfalke — APPROVED** [review]
This is looking really great.

I have sprung a concern about the highlight colors. Right now, the active chip color is the same as the active entity in the variable tree, while the inactive chip color is the same as the active variable in the variable tree. I wonder if the active chip should have the same highlighting as the active variable in the variable tree, and maybe the inactive chip can have a simple outline. I'm not confident about this, so maybe we can get some feedback from others.

**Comment by dmfalke** [comment]
I'm going to merge this. We can address highlighting as a follow-up.

**Pull request merged**